subscription-deployments/create-rg/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "rgName": {
+      "value": "GEN-UNIQUE"
+    },
+    "rgLocation": {
```

Comment:
remove this when you add the defaultValue

Reply:
removed rgLocation. rgName is still there.